library/eggdrop

```diff
@@ -1,10 +1,14 @@
 Maintainers: Geo Van O <geo@eggheads.org> (@vanosg)
 GitRepo: https://github.com/eggheads/eggdrop-docker.git
 
-Tags: 1.8, 1.8.0, develop
-GitCommit: 49321b7d9b37d7a177a9a1b9c5f14eb13df5e204
+Tags: develop
```

Comment:
I'm a little bit concerned about the expectations it might create for users of this tag to have it isolated like this (which in turn will give it a dedicated space by itself under "Supported tags" in the image description) -- ie, this isn't going to be always up-to-date with the `develop` branch from Git (which I'm afraid is the level of support this appears to convey), and is more like a point-in-time snapshot that's likely going to be updated on a semi-regular/sporadic basis.  Thoughts?